etc/skydive.yml.default

```diff
@@ -104,6 +104,8 @@ agent:
     # stats_update: 1
   metadata:
     info: This is compute node
+  tcp_len_by_seq: true
```

Comment:
I understand that your patch for tcp_len_by_seq, counting flow bytes based on the tcp len and avoid counting the retransmission. Why leaving the flow bytes as it's done today and add a TCPRetransmissionLen in TCPFlowMetric
as we can store these datas and see number of retransmitted packets and bytes in a separated TCP only statistics

I would rather prefer to have TCP stack counters like, overlap, retransmission that we could add in TCP Metric, and enable them by default. Basically add counter like wireshark TCP analysis.
Another interesting subject would be IP fragmentation

What do you think ?